runtime/JavaScript/package.json

```diff
@@ -51,13 +51,13 @@
   "exports": {
     ".": {
       "node": {
-        "types": "src/index.node.d.ts",
+        "types": "./src/antlr4/index.d.ts",
         "import": "./dist/antlr4.node.mjs",
         "require": "./dist/antlr4.node.cjs",
         "default": "./dist/antlr4.node.mjs"
       },
       "browser": {
-        "types": "src/index.web.d.ts",
+        "types": "./src/antlr4/index.d.ts",
```

Comment:
Note that this is a small lie but one that isn't new. It was already here through the top-level `types`:
https://github.com/antlr/antlr4/blob/faef88016247cf4d29d968df806481206ecde36d/runtime/JavaScript/package.json#L8

The lie is that at runtime the browser entrypoint **doesn't** expose `FileStream` export but the TS thinks that it does. Note that I advise against diverging interfaces between environments because it gets really messy when a package gets used transitively etc. It's much easier to just have the same interface/exports list regardless of the environment etc.